CHANGELOG.md

```diff
@@ -5,25 +5,56 @@ All notable changes to Zebra are documented in this file.
+- Windows is now back in the second tier of supported platforms.
+- The end-of-support time interval is set to match `zcashd`'s 16 weeks.
+- The RPC serialization of empty treestates matches `zcashd`. 
 
 ### Added
 
-- Added a page to the Zebra book describing custom Testnets ([#8636](https://github.com/ZcashFoundation/zebra/pull/8636))
+- Add an init function for a standalone `ReadStateService` ([#8595](https://github.com/ZcashFoundation/zebra/pull/8595))
+- Add new parameters for custom Testnets ([#8477](https://github.com/ZcashFoundation/zebra/pull/8477), [#8518](https://github.com/ZcashFoundation/zebra/pull/8518), [#8524](https://github.com/ZcashFoundation/zebra/pull/8524))
+- Add default constructions for several RPC method responses ([#8616](https://github.com/ZcashFoundation/zebra/pull/8616))
+- Allow custom Testnets to make peer connections and configure more parameters ([#8528](https://github.com/ZcashFoundation/zebra/pull/8528))
+- Allow configurable NU5 activation height on Regtest ([#8505](https://github.com/ZcashFoundation/zebra/pull/8505))
+
+#### Docs
+
+- Add a section on Regtest usage to the Zebra book ([#8526](https://github.com/ZcashFoundation/zebra/pull/8526))
+- Describe custom Testnets in the Zebra book ([#8636](https://github.com/ZcashFoundation/zebra/pull/8636))
+
+### Changed
+
+- Drop transactions with unpaid actions ([#8638](https://github.com/ZcashFoundation/zebra/pull/8638))
+- Lower the mandatory checkpoint height ([#8629](https://github.com/ZcashFoundation/zebra/pull/8629))
+- Use the new `zcash_script` callback API ([#8566](https://github.com/ZcashFoundation/zebra/pull/8566))
+- Put Windows in Tier 2 of supported platforms ([#8637](https://github.com/ZcashFoundation/zebra/pull/8637))
+- Remove support for starting the scanner at `zebrad` startup ([#8594](https://github.com/ZcashFoundation/zebra/pull/8594))
+- Reduce the end of support time from 20 weeks to 16 weeks ([#8530](https://github.com/ZcashFoundation/zebra/pull/8530))
+- Restore parts of the internal-miner feature for use on Regtest ([#8506](https://github.com/ZcashFoundation/zebra/pull/8506))
+
+### Fixed
+
+- Allow square brackets in network config listen addr and external addr ([#8504](https://github.com/ZcashFoundation/zebra/pull/8504))
+- Fix general conditional compilation attributes ([#8602](https://github.com/ZcashFoundation/zebra/pull/8602))
+- Update `median_timespan()` method to align with zcashd implementation ([#8491](https://github.com/ZcashFoundation/zebra/pull/8491))
+- Refactor the serialization of empty treestates ([#8533](https://github.com/ZcashFoundation/zebra/pull/8533))
+- Address port conflict issues and enable related Windows tests ([#8624](https://github.com/ZcashFoundation/zebra/pull/8624))
+- Fix a bug with trailing characters in the openapi spec method descriptions ([#8597](https://github.com/ZcashFoundation/zebra/pull/8597))
+- Stop using `displaydoc` ([#8614](https://github.com/ZcashFoundation/zebra/pull/8614))
```

Comment:
Optional: This one doesn't seem very visible for users, though it does remove a dependency, maybe that affects the binary size? (And the log messages were slightly modified).

```suggestion
```

Side note: It looks like https://github.com/yaahc/displaydoc/pull/47 has merged if we want to add this dependency back, it seems pretty convenient and might be useful when we work on https://github.com/ZcashFoundation/zebra/issues/2908 or other changes related to adding/modifying error types.